frontend/pages/_error.tsx

```diff
@@ -26,13 +26,16 @@ const CustomErrorComponent: NextPage<ErrorProps> = (props) => {
    return <NextErrorComponent statusCode={props.statusCode} />;
 };
 
-CustomErrorComponent.getInitialProps = async (contextData) => {
-   // In case this is running in a serverless function, await this in order to give Sentry
-   // time to send the error before the lambda exits
-   await Sentry.captureUnderscoreErrorException(contextData);
+CustomErrorComponent.getInitialProps = withInitialCacheValue<ErrorProps>(
+   async (contextData) => {
+      // In case this is running in a serverless function, await this in order to give Sentry
+      // time to send the error before the lambda exits
+      await Sentry.captureUnderscoreErrorException(contextData);
 
-   // This will contain the status code of the response
-   return NextErrorComponent.getInitialProps(contextData);
-};
+      // This will contain the status code of the response
+      return NextErrorComponent.getInitialProps(contextData);
+   },
+   { disabled: true }
+);
```

Comment:
Super minor: Thoughts on swapping the params for readability?

```suggestion
CustomErrorComponent.getInitialProps = withInitialCacheValue<ErrorProps>(
   { disabled: true },
   async (contextData) => {
      // In case this is running in a serverless function, await this in order to give Sentry
      // time to send the error before the lambda exits
      await Sentry.captureUnderscoreErrorException(contextData);

      // This will contain the status code of the response
      return NextErrorComponent.getInitialProps(contextData);
   }
);
```

Reply:
> Sure! I might rename `func` to be more descriptive too if you don't mind.

Totally go for it!

PS I my actual hope here was to do:

```
export default withInitialCacheValue<ErrorProps>(MyCacheValue, CustomErrorComponent)
```

But I didn't get that far last night.